labyrwa/wedding
# Color palette

Testing instructions:
- run gulp develop
- navigate to color palette page
- observe colors are varying purples and greys
